source/crud/tests/unified/updateMany-comment.yml

```diff
@@ -29,8 +29,7 @@ tests:
         object: *collection0
         arguments:
           filter: &filter { _id: 1 }
-          update: &update
-            - $set: { x: 22 }
```

Comment:
This technically wasn't an error. @comandeo was supplying a [pipeline argument](https://docs.mongodb.com/manual/reference/command/update/#update-with-an-aggregation-pipeline) for the update. And since this test requires 4.4 as a minimum server version, the driver can be assured that it's supported by the server.

That said, the `$set` here is simple enough that there's probably no reason to use a pipeline. I'll let @comandeo chime in, but I think this is probably reasonable to change to a non-pipeline parameter here since we are already making other spec test changes in the PR (I wouldn't consider this worth it alone, though).